src/components/MidiInput.tsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-console */
+import { useEffect } from 'react'
+import { WebMidi, Input } from 'webmidi'
+import useStore from '../store/useStore'
+
+const MIDIListener = () => {
+  const scenes = useStore((state) => state.scenes)
```

Comment:
Need to call if new scene edited or added. Can I have some help on how to do this?